draft-ietf-cbor-7049bis.md

```diff
@@ -408,27 +409,31 @@ floats as integers or vice versa, perhaps to save encoded bytes.
 
 A CBOR data item  ({{cbor-data-models}}) is encoded to or decoded from
 a byte string as described in this section.  The encoding is
-summarized in {{jumptable}}.
+summarized in {{jumptable}}.  An encoder MUST produce only well-formed
+items.  A decoder MUST stop and return an error with no data when it
```

Comment:
Again, programs that decode CBOR-like inputs with some diagnostic capabilities (or even some form of repair) are desirable.  So we should, again, define a CBOR decoder as one that only accepts well-formed input.

Reply:
With the caveat that "an error" can include arbitrary diagnostic information, I think that's the effect of writing this MUST. I agree that this class of programs-that-are-not-CBOR-decoders is useful.